test/suites/integration/defined_target_ports_test.go

```diff
@@ -81,15 +81,15 @@ func performVerification(service *v1.Service, deployment *appsv1.Deployment, def
 	targetGroup := testFramework.GetTargetGroup(ctx, service)
 	Expect(*targetGroup.VpcIdentifier).To(Equal(os.Getenv("CLUSTER_VPC_ID")))
 	Expect(*targetGroup.Protocol).To(Equal("HTTP"))
-
-	targets := testFramework.GetTargets(ctx, targetGroup, deployment)
```

Comment:
testFramework.GetTargets() Try to get either TargetStatusInitial or TargetStatusHealthy status targets, however, this test file use `testFramework.NewHttpRoute()` in which has:

```
Spec: v1beta1.HTTPRouteSpec{
			CommonRouteSpec: v1beta1.CommonRouteSpec{
				ParentRefs: []v1beta1.ParentReference{{
					Name: v1beta1.ObjectName(parentRefsGateway.Name),
					//SectionName: lo.ToPtr(v1beta1.SectionName("http")),
				}},
			},
			Rules: rules,
		},
```

No listener means it will never setup healthcheck. the target status always be `TargetStatusUnused`